src/changes/changes.xml

```diff
@@ -33,6 +33,9 @@
       <action issue="LOG4J2-3550" dev="rgoers" type="fix" due-to="DongjianPeng">
         SystemPropertyAribiter was assigning the value as the name.
       </action>
+      <action issue="LOG4J2-3560" dev="schlosna" type="fix">
```

Comment:
I'll take care of this update prior to merging, the dev attribute reflects the committer who has signed off on the change while `due-to` provides attribution for the work if it wasn't generated by the committer.
```suggestion
      <action issue="LOG4J2-3560" dev="ckozak" type="fix" due-to="David Schlosnagle">
```

Reply:
Sounds good!